contentcuration/contentcuration/frontend/shared/views/Sandbox.vue

```diff
@@ -1,25 +1,72 @@
 <template>
 
-  <div>
-    Insert component here for testing!
-    <TemplateComponent />
-  </div>
+  <VApp>
```

Comment:
It's mostly just an entry point for testing components both locally and on the demo servers since there's still a lot of work needed on the page it's going to be moved to